docker/wait-for-db.sh

```diff
@@ -18,15 +18,18 @@ set -eu
 
 host="${1}"
 port="${2}"
+timeout="${3:-20}"
+NEXT_WAIT_TIME=0
 
-until PGPASSWORD="${POSTGRES_PASSWORD}" psql \
+until [ PGPASSWORD="${POSTGRES_PASSWORD}" psql \
         --host="${host}" \
         --port="${port}" \
         --username "${POSTGRES_USER}" \
         --dbname "${POSTGRES_DB}" \
-        --command '\q' > /dev/null 2>&1; do
+        --command '\q' > /dev/null 2>&1 || $NEXT_WAIT_TIME -eq "${timeout}" ]; do
   echo "Waiting for postgres to become available..."
   sleep 1
+  let NEXT_WAIT_TIME=NEXT_WAIT_TIME+1
```

Comment:
We can simply the timeout logic by decrementing the `timeout` variable by `1`. For example,

```
sleep 1
timeout=$(( timeout-1 ))
```

This eliminates the need to define the extra variable `NEXT_WAIT_TIME`. Now, on each connection attempt, we simply check:

 ```
[[ ... || $timeout -eq 0 ]]
```